tests/sources/rest_api/configurations/source_configs.py

```diff
@@ -395,6 +395,18 @@ def repositories():
             repositories(),
         ],
     },
+    {
+        "client": {"base_url": "https://github.com/api/v2"},
+        "resources": [
+            {
+                "name": "issues",
+                "endpoint": {
+                    "path": "dlt-hub/{repository}/issues/",
```

Comment:
Looks like this line is causing the tests to fail as `{repository}` path parameter is missing.
```suggestion
                    "path": "user/repos",
```

Reply:
ah, thank you!